src/script/entity/Conversation.ts

```diff
@@ -374,10 +375,13 @@ export class Conversation {
 
     this.messages_unordered = ko.observableArray();
     this.messages = ko.pureComputed(() =>
-      this.messages_unordered().sort((message_a, message_b) => {
-        return message_a.timestamp() - message_b.timestamp();
-      }),
+      this.messages_unordered()
+        .slice()
```

Comment:
i think i prefer [...this.messages_unordered()] in this scenario, more readable to me

Reply:
*suggestion